validator/node/node.go

```diff
@@ -579,30 +584,20 @@ func Web3SignerConfig(cliCtx *cli.Context) (*remoteweb3signer.SetupConfig, error
 		}
 
 		if publicKeysSlice := cliCtx.StringSlice(flags.Web3SignerPublicValidatorKeysFlag.Name); len(publicKeysSlice) > 0 {
-			pks := make([]string, 0)
 			if len(publicKeysSlice) == 1 {
 				pURL, err := url.ParseRequestURI(publicKeysSlice[0])
 				if err == nil && pURL.Scheme != "" && pURL.Host != "" {
 					web3signerConfig.PublicKeysURL = publicKeysSlice[0]
 				} else {
-					pks = strings.Split(publicKeysSlice[0], ",")
-				}
-			} else if len(publicKeysSlice) > 1 {
-				pks = publicKeysSlice
-			}
-			if len(pks) > 0 {
-				pks = slice.Unique[string](pks)
-				var validatorKeys [][48]byte
-				for _, key := range pks {
-					decodedKey, decodeErr := hexutil.Decode(key)
-					if decodeErr != nil {
-						return nil, errors.Wrapf(decodeErr, "could not decode public key for web3signer: %s", key)
-					}
-					validatorKeys = append(validatorKeys, bytesutil.ToBytes48(decodedKey))
+					web3signerConfig.ProvidedPublicKeys = strings.Split(publicKeysSlice[0], ",")
 				}
-				web3signerConfig.ProvidedPublicKeys = validatorKeys
+			} else {
+				web3signerConfig.ProvidedPublicKeys = publicKeysSlice
```

Comment:
If we are doing `web3signerConfig.ProvidedPublicKeys = strings.Split(publicKeysSlice[0], ",")` just abve, should we not be doing it here too

Reply:
no, that case is only to handle the situation of having 1 element and not having it separated correctly. if it's more than 0 and the slice is not exactly 1 then it has already split

Reply:
Ah thank you, you're right, I didn't follow it properly